packages/calling/src/Contacts/ContactsClient.ts

```diff
@@ -105,9 +102,6 @@ export class ContactsClient implements IContacts {
   /**
    * Encrypt emails, phoneNumbers, sipAddresses.
    *
-   * @param contactDetails - Array ContactDetail.
-   * @param encryptionKeyUrl - KMS encryptionKeyUrl for the contact.
```

Comment:
Need those parameters and descriptions back

Reply:
Same comment as below for private methods